src/scss/components/optional/_order-card.scss

```diff
@@ -68,6 +82,31 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
         }
     }
 
+    .c-orderCard-image--noOpenRestaurants {
+        background-color: $grey--lightest;
+    }
+
+    .c-orderCard-iconContainer {
+        width: 64px;
+        height: 64px;
+        border-radius: 50%;
+        background-color: white;
+        position: absolute;
```

Comment:
Minor one but it would be nice to combine some of the repeat styles.

```
.c-orderCard-iconContainer,
svg {
  position: absolute;
  top: 50%;
  right: 50%;
  transform: translate(50%, -50%);
}
```

Reply:
I moved them to utility class instead